api/src/main/java/org/openmrs/module/inventorypoc/batch/validation/BatchValidator.java

```diff
@@ -22,4 +27,20 @@ public void validateAdjustments(final Batch batch, final Double newValue) {
 			    Context.getLocale()));
 		}
 	}
+	
+	public void validateWaste(final DrugOrder drugOrder, final Location location, final Double quantity,
+	        final Date date) {
+		
+		final Double currentStockBalance = Context.getService(BatchService.class)
```

Comment:
For future reference to the services classes, be initailizing them in the constructors which can be easly be proted to test classes. For example
private BatchService batchService;
public BatchValidatior(BatchService batchService){
this.batchService = batchService
}
then just use batchService as an instance **That is for later use**